### ISSUE TITLE
US County Completeness in county.fips table

### ISSUE THREAD

**Issue by mta614** [adeckmyn/maps#11]
Hi there,

Noticing an issue with some counties not appearing when visualizing US Counties. The county that caught my attention was Oglala Lakota County, in South Dakota (STATECOUNTYFIPS: 46102), but given that there are 3143 counties in the Census ACS data and only 3085 in the maps county.fips table, I imagine others are also missing.

**Comment by adeckmyn** [comment]
Thanks for noticing this. In fact, Oglala Lakota is not missing, it is just listed with its old name (Shannon, fips 46113) that was changed recently (in 2015 according to Wikipedia). The difference in number of counties is due to Alaska and Hawaii not being part of the map (and thus not in the data table), plus the cities in Virgina, which are not given as polygons in the map, and thus are also not in the data table. Upon checking with 2017 census data, Oglala Lakota <-> Shannon appears to be the only recent 
change that was not yet updated.

**Comment by angela-li** [comment]
Following up on this @mta614 - if you need to grab the FIPS code for counties in Alaska or Hawaii, the [tigris package](https://github.com/walkerke/tigris) by Kyle Walker has a [`fips_code` function](https://www.rdocumentation.org/packages/tigris/versions/0.7/topics/fips_codes) that's complete.

@adeckmyn - Is there any plan for adding the Alaska and Hawaii counties into the fips code table?

**Comment by adeckmyn** [comment]
not really. The county-fips data set is partnered to the 'county' map data base, which does not include them. In a legacy package like 'maps' it is always risky to change anything after (for the basic data) 20 years. I would have to see if adding counties to the data set could affect any application for the 'county' map. for the same reason, simply updating Oglala Lakota is less simple than just changing the name and fips code. All pre-2015 data sets would suddenly no longer fit the county map. It will have to happen, but carefully.
